packages/graphics/opengl-meson-aarch64/system.d/unbind-console.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Unbind framebuffer console
+
+ConditionPathExists=/sys/class/vtconsole/vtcon1/bind
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c 'echo 0 > /sys/class/vtconsole/vtcon1/bind'
+
+[Install]
+WantedBy=graphical.target
```

Comment:
FWIW this process of unbinding is completely generic to DRM drivers, even for the nvidia (non-legacy) driver.  
Perhaps it's worth reworking and dropping the duplicated boilerplate at some point.